lib/client/db/dbcmd/dbcmd.go

```diff
@@ -395,5 +413,33 @@ func (c *cliCommandBuilder) getSQLServerCommand() *exec.Cmd {
 		args = append(args, "-d", c.db.Database)
 	}
 
-	return exec.Command(mssqlBin, args...)
+	return c.exe.Command(mssqlBin, args...)
+}
+
+type connectionCommandOpts struct {
+	localProxyPort int
+	localProxyHost string
+	caPath         string
+	noTLS          bool
+}
+
+type ConnectCommandFunc func(*connectionCommandOpts)
+
+func WithLocalProxy(host string, port int, caPath string) ConnectCommandFunc {
```

Comment:
Nit: Since you're here anyway, could you add godocs on WithLocalProxy and ConnectCommandFunc? I should've done it when adding WithNoTLS below...